dora/tests/integration/src/test/java/alluxio/client/fs/io/FileOutStreamIntegrationTest.java

```diff
@@ -184,7 +193,8 @@ public void longWrite() throws Exception {
    */
   @Test
   public void outOfOrderWrite() throws Exception {
-    AlluxioURI filePath = new AlluxioURI(PathUtils.uniqPath());
+    String uniqPath = PathUtils.uniqPath();
+    AlluxioURI filePath = new AlluxioURI(mRoot).join(uniqPath);
```

Comment:
BTW, as jiacheng said, if `PathUtils.uniqPath();` generates a directory, I don't think it is a good idea to use its return value as a file path.